it/topics/scripting_languages/JavaScript/frameworks.md

```diff
@@ -0,0 +1,393 @@
+        - [React: Getting Started](https://app.pluralsight.com/library/courses/react-js-getting-started/table-of-contents)
+        - [React Fundamentals](https://app.pluralsight.com/library/courses/react-fundamentals-update/table-of-contents)
+        - [Advanced React.js](https://app.pluralsight.com/library/courses/reactjs-advanced/table-of-contents)
+    - [Scotch.io](https://scotch.io/) (corsi gratuiti)
+        - [Introduzione a React](https://scotch.io/starters/react/getting-started-with-react-2019-edition?ref=home-start-here)
+
+
+### React root
+
+React viene eseguito su un singolo file `index.html` con un` div` come questo. A volte basta "root".
+
+```html
+<div id="react-root">
+</div>
+```
+
+Sul sito di Instagram, troverai qualcosa del genere sul loro sorgente HTML:
+
+Se disabiliti JavaScript e ricarichi la pagina, vedrai solo il logo di Instagram, poiché tutto il resto è reso da JavaScript.
+
+
+### Hello world
+
+```html
+<div id="root">
+</div>
+
+<script>
+    ReactDOM.render(
+        <h1>Hello, world!</h1>,
+        document.getElementById('root')
+    );
+</script>
+```
+
+Questo utilizza la sintassi JSX, quindi avrai bisogno della libreria React e di Babel internamente per convertirla in semplice JavaScript ed eseguirla.
+
+
+### Routing
+
+Se aggiungi la libreria di React Router puoi gestire facilmente il passaggio tra i percorsi del browser. Ecco una guida rapida per questo.
+
+```javascript
+import React from "react";
+import ReactDOM from "react-dom";
+import { BrowserRouter } from "react-router-dom";
+
+function App() {
+  return <h1>Hello, world! This is React Router</h1>;
+}
+
+ReactDOM.render(
+  <BrowserRouter>
+    <App />
+  </BrowserRouter>,
+  document.getElementById("root")
+);
+```
+
+
+## React Native
+
+### Links
+
+- [Sito ufficiale](https://facebook.github.io/react-native/).
+    > Crea app native per Android e iOS usando React.
+     > Scritto in JavaScript - reso con codice nativo.
+     > React Native è come React, ma utilizza componenti nativi anziché componenti Web come blocchi predefiniti.
+- [Introduzione](https://facebook.github.io/react-native/docs/getting-started)
+    > Questa pagina ti aiuterà a installare e creare la tua prima app React Native.
+- Tutorials
+    - [Impara le basi](https://facebook.github.io/react-native/docs/tutorial).
+- [Github repo](https://github.com/facebook/react-native)
+
+### Hello world
+
+```javascript
+import React, { Component } from 'react';
+import { Text, View } from 'react-native';
+
+export default class HelloWorldApp extends Component {
+    render() {
+        return (
+        <View style={{ flex: 1, justifyContent: "center", alignItems: "center" }}>
+            <Text>Hello, world!</Text>
+        </View>
+        );
+    }
+}
+```
+
+
+## Electron
+
+- [Homepage ufficiale](https://www.electronjs.org/)
+	-> Crea app desktop multipiattaforma con JavaScript, HTML e CSS
```

Comment:
```suggestion
	- > Crea app desktop multipiattaforma con JavaScript, HTML e CSS
```

Reply:
This is a bullet point with a quote inside. Your view makes it render into a single line which is not nice.

`[Homepage ufficiale](https://www.electronjs.org/) -> Crea app desktop...`